### PR TITLE
GH-970 Adds HTML5 color input to form/Input.js

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes GH-970

@GordonSmith @mlzummo Please review

http://rawgit.com/jbrundage/Visualization/GH-970-PE-Color/demos/dermatology.html?src/layout/Surface

Signed-off-by: Jay Brundage <jaman.brundage@lexisnexis.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
why not just pass the correct html5 value for the input field when it sees html-color have it pass "color" ?

**Comment by mlzummo** [comment]
unless you displaying 2 textbox's one with the value and 1 with the color rep?

File: src/form/Input.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
     Input.prototype._class += " form_Input";
     Input.prototype.implements(IInput.prototype);
 
-    Input.prototype.publish("type", "text", "set", "Input type", ["textbox", "number", "checkbox", "button", "select", "textarea", "date"]);
+    Input.prototype.publish("type", "text", "set", "Input type", ["textbox", "number", "checkbox", "button", "select", "textarea", "date", "html-color"]);
```
Comment (mlzummo) [inline src/form/Input.js]: remove textbox ...

**Comment by jbrundage** [comment]
@GordonSmith @mlzummo @dtsnell4 What can be done about these Travis CI failures (Seems to be related to Wordcloud?)

**Comment by mlzummo** [comment]
It's already fixed see my fixed prs

@jbrundage

**Comment by jbrundage** [comment]
@mlzummo So it's fixed but needs to be merged?

**Comment by mlzummo** [comment]
@jbrundage Yes take a look at any of the outstanding PR's that have passed and they have the fix .. hopefully i did it correctly ...

**Comment by mlzummo** [comment]
@jbrundage can you rebase this on my input changes ?

**Comment by GordonSmith** [comment]
Looks fine.

**Pull request merged**